src/WebSDL/settings/base.py

```diff
@@ -58,7 +58,7 @@
     'requests',
 ]
 
-MIDDLEWARE_CLASSES = [
+MIDDLEWARE = [
```

Comment:
FYI: `MIDDLEWARE_CLASSES` is a thing of pre-Django 1.10. From Django v1.10 onward, this changed to `MIDDLEWARE`.

Reply:
nice!